client/src/components/DataFiles/DataFilesStatus/DataFilesStatus.js

```diff
@@ -1,4 +1,4 @@
-import { findSystemDisplayName } from 'utils/systems';
+import { findSystemOrProjectDisplayName } from 'utils/systems';
```

Comment:
Need to resolve this with:

https://github.com/TACC/Frontera-Portal/blob/4810abdd43e9e7380aa0771034160df9a8608791/client/src/utils/systems.js#L30

Reply:
> @jchuahtacc @Strmiska What's the resolution path for this PR?

I think we should merge this PR. It shows the paths correctly in the notification, but I think some more time needs to be spent reevaluating how we should check for the system 'scheme' to determine what to display. Should we expect the 'scheme' to be provided everywhere we try to use this utility, or should the utility find another way to display the correct name of a system?

**The param in question:**
https://github.com/TACC/Core-Portal/pull/320/files#diff-5f667ae20b2b798ee1855adb1d4e5aef4364ab6667233bc1d8652cef8d3b0ca7R45

Reply:
@Strmiska , can you update from `main`?  the displayed diff is showing changes that are already in main so an update would help reviewers.

Reply:
In the process of that now!